playbooks/roles/deploy-osh/tasks/enroll-caasp-nodes-for-osh.yml

```diff
@@ -45,24 +45,29 @@
     - _enrollforopenstack.rc != 0
     - _enrollforopenstack.stderr.find("already has a value (enabled)") == -1
   with_items:
-    - "{{ _all_casp_nodenames.stdout_lines }}"
+    - "{{ _all_caasp_nodenames.stdout_lines }}"
 
 - name: Store all nodes details
   command: kubectl get nodes -o json
   changed_when: false
   register: _nodes
 
 - name: Find master node
-  set_fact:
-    master_node: "{{ _nodes.stdout_lines | join('') | from_json | json_query(query) }}"
-  vars:
-    query: 'items[].spec.{node: externalID, taints: taints} | [?taints].node'
+  shell: |
+    set -o pipefail
```

Comment:
is pipefail needed here? There is no pipes in the command

Reply:
You are right, but I'll just go ahead and merge it anyway...